src/app/components/configure-app/configure-app.component.ts

```diff
@@ -104,12 +104,24 @@ export class ConfigureAppComponent implements OnInit {
 
   powOptions = [
     { name: 'Best Option Available', value: 'best' },
-    { name: 'Client Side - WebGL [Recommended] (Chrome/Firefox)', value: 'clientWebGL' },
-    { name: 'Client Side - CPU', value: 'clientCPU' },
-    { name: 'Server - Nault Server', value: 'server' },
+    { name: 'Client Side - GPU/WebGL', value: 'clientWebGL' },
+    { name: 'Client Side - CPU (Slowest)', value: 'clientCPU' },
+    { name: 'Selected Server - dPoW / boomPoW / Other', value: 'server' },
```

Comment:
For consistency with "Client Side":
```suggestion
    { name: 'Server Side - dPoW / boomPoW / Other', value: 'server' },
```

Are the possible PoW mechanisms relevant "dPoW / boomPoW / Other" ?

Reply:
Solved